extensions/variable-width-instruction-standard.md

```diff
@@ -16,4 +16,10 @@ Instruction prefixes are a sequence of one or more bytes which modify the instru
 
 ### Defined Prefix Sequences
 
-`110x xxxx`
+
+| Prefix      | Description        |
+|:------------|:-------------------|
+| `1010 xxxx` | Unused             |
+| `1100 xxxx` | Expanded registers |
```

Comment:
This could link to the corresponding document

Reply:
Possibly. I figured the description would say what uses it. Otherwise it can be put in the VWI standard to keep track of what's a prefix and what isn't as well as what is still unused from the ISA